app/vep/utils/nextflow.py

```diff
@@ -0,0 +1,21 @@
+import requests
+import json
+from core.config import NF_TOKEN, NF_PIPELINE_URL, NF_COMPUTE_ENV_ID, NF_WORK_DIR, SEQERA_WORKFLOW_LAUNCH_URL
+from models.pipeline_model import PipelineParams
+
+def launch_workflow(pipeline_params: PipelineParams):
+    headers = {
+        "Content-Type": "application/json",
+        "Authorization": f"Bearer {NF_TOKEN}"
+    }
+
+    response = requests.post(SEQERA_WORKFLOW_LAUNCH_URL, headers=headers, json=pipeline_params)
+
+    # Check if successful
+    if response.status_code == 200:
+        return response.json()
+    else:
+        print(f"Failed to launch workflow. Status code: {response.status_code}")
+        return response.text
```

Comment:
Shouldn't the function raise some kind of error if the request to the platform failed? If I am reading the code correctly, all errors will be quietly swallowed.

Reply:
As I said earlier, error handling is not required in this ticket, but since you mentioned I will add it.

Reply:
hmm it makes sense for the launch_workflow to just raise an error and the caller handles it accordingly. Lets discuss in standup